api/controllers/__init__.py

```diff
@@ -3,3 +3,4 @@
 from .users import *
```

Comment:
There are a couple of files listed here that I don't see in the "changed files" sidebar for the PR, so I just wanted to make sure the files still exist:

L1: `from .default import *`
L3: `from .users import *`